pkg/cloudprovider/providers/gce/gce_disks.go

```diff
@@ -809,6 +862,48 @@ func (gce *GCECloud) DeleteDisk(diskToDelete string) error {
 	return err
 }
 
+func (gce *GCECloud) ResizeDisk(diskToResize string, oldSize resource.Quantity, newSize resource.Quantity) (resource.Quantity, error) {
+	disk, err := gce.GetDiskByNameUnknownZone(diskToResize)
+	if err != nil {
+		return oldSize, err
+	}
+
+	requestBytes := newSize.Value()
+	// GCE resizes in chunks of GBs (not GiB)
+	requestGB := volume.RoundUpSize(requestBytes, 1000*1000*1000)
```

Comment:
Can you make GB a constant? Do we have one already defined somewhere?

Reply:
`RoundUp` is for rounding a value upto certainc scale not rounding the value to GB or GiB chunks.

Reply:
@msau42 lets do the GB constant cleanup in a follow up PR, because I did not find a pre-defined GB constant and such a change will affect other volume types too. I will open a follow up github issue for myself.